modules/mono/mono_gd/gd_mono_utils.h

```diff
@@ -151,6 +154,21 @@ struct ScopeThreadAttach {
 	MonoThread *mono_thread;
 };
 
+template <typename... P>
```

Comment:
Is this C++11/14 compatible in the end or does this form of parameter pack require C++17? https://en.cppreference.com/w/cpp/language/parameter_pack

Reply:
I think it's C++11. I only had to look for an alternative for the `if constexpr`s when backporting this.